code/modules/admin/verbs/shuttles.dm

```diff
@@ -0,0 +1,108 @@
+/proc/emergency_sanity_check()
+	if(SSshuttle.emergency.mode != SHUTTLE_IDLE)
+		var/confirm = alert(src, "Modification of the emergency shuttle while it is not idle can be highly dangerous, and may result in WEIRD UNPREDICTABLE SHIT. Are you SURE you want to continue? Obviously if you're not touching the emergency shuttle, then you're probably fine.", "Confirm", "Yes", "No")
+		if(confirm == "Yes")
+			return TRUE
+		else
+			return FALSE
+	else
+		return TRUE
+
+/client/proc/cmd_admin_destroy_shuttle()
+	set category = "Admin"
+	set name = "Shuttle Destroy"
+
+	if (!holder)
+		src << "Only administrators may use this command."
+		return
+
+	if(!emergency_sanity_check())
+		return
+
+	var/list/names = list()
+	var/obj/docking_port/mobile/M
+	for (var/atom/AM in SSshuttle.mobile)
+		M = AM
+		names += M.name
+
+	var/selected = input("Select shuttle to DESTROY", "Shuttles") in names
+
+	var/decide_against_msg = "You decide against destroying a shuttle."
+
+	if(!selected)
+		src << decide_against_msg
+		return
+
+	var/confirm = alert(src, "Are you sure you want to destroy [selected]?", "Confirm", "Yes", "No")
+
+	if(confirm != "Yes")
+		src << decide_against_msg
+		return
+
+	var/destroyed = FALSE
+	for (var/atom/AM in SSshuttle.mobile)
+		M = AM
+		if(M.name == selected)
+			M.jumpToNullSpace()
+			destroyed = TRUE
+			break
+
+	if(!destroyed)
+		src << "<span class='warning'>Something went wrong, the selected shuttle doesn't exist anymore."
+		return
+
+	log_admin("ShuttleDestroy: [M]")
+	message_admins("<span class='adminnotice'>[key_name_admin(usr)] has used <b>ShuttleDestroy on [selected]</b><BR></span>")
+	feedback_add_details("admin_verb","SHTDEL") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+
+/client/proc/cmd_admin_import_shuttle()
+	set category = "Admin"
+	set name = "Shuttle Import"
+
+	if (!holder)
+		src << "Only administrators may use this command."
+		return
+
+	if(!emergency_sanity_check())
+		return
+
+	var/datum/map_template/template
+	// TODO would be helpful if it only highlighted actual shuttles rather
+	// than ALL OF THEM
+	var/map = input(usr, "Choose a Shuttle Template to import","Import Shuttle Template") as null|anything in map_templates
```

Comment:
Why not just populate a list of only shuttle templates like how space and lava ruins ones do?